tests/materialize/materialize-databricks/snapshot.json

```diff
@@ -800,6 +809,17 @@
   ],
   "table": "`main`.`some-schema`.formatted_strings"
 }
+{
+  "rows": [
+    {
+      "flow_document": "{\"_meta\":{\"uuid\":\"fb0cc8b0-20e1-11ee-990b-ffd12dfcd47f\"},\"id\":1,\"unsigned_bigint\":18446744073709551615}",
+      "flow_published_at": "2023-07-12T18:29:04.671352Z",
+      "id": 1,
+      "unsigned_bigint": 18446744073709552000
```

Comment:
This inaccuracy seems to be due to `jq` not supporting unsigned uint64s properly... I've verified the value comes correctly from our `fetch-data`, but gets clamped by `jq`